labs/lab18-helm-charts/README.md

```diff
@@ -438,8 +436,8 @@ replicated release create --auto --promote lab18-helm-charts -y
 
 Now that we have a new release available, let's update our deployed application to test the change.
 
-Click on **Check for Update** and click on **Deploy** once available. Once deployed, you should be able to browse to port `8080` and see Wordpress.
-<img width="880" alt="Screen Shot 2022-07-10 at 3 02 51 PM" src="https://user-images.githubusercontent.com/3730605/178160342-bb7c897d-1baa-46f4-a5d1-1503921d50de.png">
+Click on **Check for Update** and click on **Deploy** once available. Once deployed, you should be able to browse to port `30808` and see Wordpress.
+<p align="center"><img src="img/wordpress-blog.png" alt="Wordpress Blog" width="600" margin=auto/></img></p>
```

Comment:
Is the Wordpress Blog page for the right version we are checking out? I thought it look diferent

Reply:
Looks the same to me. This is the original one: https://user-images.githubusercontent.com/3730605/178160342-bb7c897d-1baa-46f4-a5d1-1503921d50de.png
What is different?